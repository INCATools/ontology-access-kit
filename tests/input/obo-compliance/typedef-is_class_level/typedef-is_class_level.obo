format-version: 1.4
ontology: typedef-is_class_level

[Typedef]
id: R:1
is_class_level: true

