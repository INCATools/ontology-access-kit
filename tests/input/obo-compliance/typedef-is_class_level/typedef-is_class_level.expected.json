{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/typedef-is_class_level.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/R_1",
      "type" : "PROPERTY",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#is_class_level",
          "val" : "true"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}